_inc/client/apps/style.scss

```diff
@@ -160,50 +160,55 @@
 	}
 }
 
-.jp-landing-apps__feature-col {
+.jp-landing-apps__feature {
 
 	@include breakpoint( ">660px" ) {
-		flex-basis: 50%;
 
-		@include breakpoint( ">660px" ) {
```

Comment:
Nah. I just think looking at the diff within the browser makes it look that way.